src/aws.ts

```diff
@@ -504,6 +507,237 @@ export namespace $AWS {
       },
     });
   }
+
+  export const SDK: StepFunctionSDK = new Proxy<any>(
+    {},
+    {
+      get(_, serviceName: ServiceKeys) {
+        return new Proxy<any>(
```

Comment:
@sam-goodwin I am a little worried that the closure serializer will catch the proxy and fail. Need to test this at least.

Reply:
Yeah I thought about that but it should only be serializing the returned data from the `get`. I think it'll work but of course need a test.

Reply:
It matters what ends up being serialized. Its pretty easy with the current serializer for it to try to serialize everything.